llvm/lib/Target/AMDGPU/BUFInstructions.td

```diff
@@ -2499,7 +2494,7 @@ multiclass MUBUF_Real_AllAddr_gfx11_gfx12_Renamed<bits<8> op, string real_name>
 
 class MUBUF_Real_Atomic_gfx11_impl<bits<8> op, string ps_name,
```

Comment:
Looks like the next thing to get rid of?

Reply:
Yes, I am working on a larger patch to remove some useless `*_impl` classes.